Assets/SteamVR_Unity_Toolkit/Scripts/Abstractions/VRTK_WorldPointer.cs

```diff
@@ -97,7 +98,12 @@ protected virtual void InitPointer()
 
         protected virtual void SetPlayAreaCursorTransform(Vector3 destination)
         {
-            playAreaCursor.transform.position = destination;
+            playArea = GameObject.FindObjectOfType<SteamVR_PlayArea>();
+            hmdCamera = GameObject.FindObjectOfType<SteamVR_Camera>();
```

Comment:
Use the VRTK.DeviceFinder.HeadsetTransform() method for getting the hmd as it handles the differences between unity 5.3 and 5.4

the method is: https://github.com/thestonefox/SteamVR_Unity_Toolkit/blob/master/Assets/SteamVR_Unity_Toolkit/Scripts/Helper/DeviceFinder.cs#L35

so this line would be (with a null check as well for performance):

```
if (hmdCamera == null) {
  hmdCamera = DeviceFinder.HeadsetTransform();
}
```